phpstan.neon

```diff
@@ -1,10 +1,9 @@
 parameters:
+    level: 3
```

Comment:
level: 7 was removed above, should this match what was removed?

Reply:
No, as mentioned in the PR title, I lowered the PHPStan level to 3 on this component, as level 7, under the more recent PHPStan version, caused a lot of errors to show up that I'm not able to take the time to resolve (probably ignore in most cases) right now.